packages/python-packages/doc-warden/warden/enforce_changelog_content.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from __future__ import print_function
+
+import os
+import markdown2
+import bs4
+import re
+from .warden_common import get_omitted_files
+from docutils import core
+from docutils.writers.html4css1 import Writer,HTMLTranslator
```

Comment:
During content verification, I _believe that I parsed the RST to HTML already. Why was this change for conversion necessary? Lack of details that you needed?